index.css

```diff
@@ -0,0 +1,141 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body, html{
+    font-family: "Inter", sans-serif;
+    font-optical-sizing: auto;
+    font-style: normal;
+}
+
+body{
+    background-color: hsl(47, 88%, 63%);
+    margin: 0 auto;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
+}
+
+.container{
+    max-width: 384px;
+    max-height: 522px;
+    border: 1px solid black;
+    padding: 24px 24px;
+    border-radius: 20px;
+    box-sizing: border-box;
+    background-color: #ffff;
+    margin-bottom: 10px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+}
+
+
+.illustration-image{
+    width: 100%;
+    width: 336px;
+    height: 200px;
+}
+
+.illustration-image img{
+    border-radius: 20px;
+    width: 100%;
+    height: 100%;
+    display: block;
+    object-fit: cover;
+}
+
+.header{
+    width: 336px;
+    height: 194px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    
+}
+
+.header button{
+    display: block;
+    padding: 3px 7px ;
+    background-color: hsl(47, 88%, 63%);
+    border: none;
+    font-weight: 600;
+    margin-bottom: 10px;
+    border-radius: 3px;
+    cursor: pointer;
+}
+
+header button:hover,
+.header button:focus {
+    background-color: hsl(47, 88%, 53%);
+    outline: none;
+}
+
+.header time{
+    font-size: 15px;
+    color: hsl(0, 0%, 7%);
+}
+
+.header h1{
+    margin: 15px 0;
+    font-size: 20px;
+}
+
+.header h1:hover{
+    color:hsl(47, 88%, 63%);
+    transition: 0.3s ease;
+    cursor: pointer;
+}
+
+.author-image{
+    display: flex;
+    justify-content:flex-start;
+    align-items: center;
+    gap: 10px;
+}
+
+article .language-description{
+    font-size: 16px;
+    color:  hsl(0, 0%, 42%);
+    line-height: 1.5;
+}
+
+.author-image figcaption{
+    font-size: 12px;
+    font-weight: 600;
+    color: hsl(0, 0%, 7%);
+}
+
+
+.attribution { font-size: 11px; text-align: center; }
+.attribution a { color: hsl(228, 45%, 44%); }
+.attribution a{text-decoration: underline;}
+
+
+/* Media query for screens ≥ 375px */
+@media screen and (min-width: 375px){
```

Comment:
I have noticed overall that the component seems small on desktop but when on mobile preview, the width is larger than that of desktop. Make changes so that maybe on desktop, the width will be larger than that of mobile.. See the design again for this changes.